woocommerce/google-listings-and-ads
# Accept TOS when creating a new Ads account

### Changes proposed in this Pull Request:

Previously we decided to have only one TOS accepted popup, this now needs to be done for both Merchant Center and Ads when creating new accounts. So this PR allows us to accept the TOS for each service individually. On the frontend the user is only able to create the new account when accepting the TOS, so when creating the account we record the acceptance.

Closes #159

When creating the [Merchant AccountController](https://github.com/woocommerce/google-listings-and-ads/issues/89) we will need to call the same code with `google-mc` to accept the TOS.

### Detailed test instructions:

1. Setup and test a create account request as described in https://github.com/woocommerce/google-listings-and-ads/pull/141
2. Check the DB table `wcc_tos` in your local WCS recorded an entry for the service `google-ads`
3. If we create multiple accounts it will record multiple entries in the DB table to keep track of the history


### Changelog Note:
- Record TOS acceptance when creating a new Ads account
